frontend/src/i18n/locales/de/translation.json

```diff
@@ -315,6 +315,15 @@
   "Default Request": "Standardanforderung",
   "Max": "Max",
   "Min": "Min",
+  "Invalid namespace name: name must be 63 characters or shorter": "",
+  "Invalid namespace name: name must start with an alphanumeric character": "",
+  "Invalid namespace name: name must end with an alphanumeric character": "",
+  "Invalid namespace name: name must consist of lower case alphanumeric characters or -": "",
```

Comment:
Can you reuse the string we already have for SettingsCluster? So we don't have to translate these? (We are missing the 63 chars one, so maybe keep that or adapt the new one)

Reply:
fixed!